meeting-notes/2022-05-14.md

```diff
@@ -0,0 +1,261 @@
+# 5/14/22
+
+@pcohen, @rntz
+
+## TODOs
+
+- [ ] Phil JSON:
+  - add cursor selection ranges to JSON output
+- [ ] rntz JSON:
+  - Add selections (including the anchor)
+  - Multiple cursors
+  - Buffer filename should be null if nonexistent
+  - camelCase
+
+@rntz sez: "maybe we should put all these files in a ~/.cursorless/ directory?"
```

Comment:
you could have a look at what we do for command [server](https://github.com/pokey/command-server/blob/050af7dc4a03df14002f5e8df13296b6607ac45d/src/paths.ts#L4-L12) and [client](https://github.com/pokey/talon-vscode-command-client/blob/1f5b37394c21e015a3d92dfbb804903d1d142cf2/command_client.py#L211-L224)